### PR TITLE
Newsweek homepage

### PR DESCRIPTION
**Pull request by chirchir12**
## Newsweek clone project
- to demonstrate my understanding using bootstrap framework

### CODE REVIEW

**Comment by chirchir12** [comment]
## implemented homepage
* this is a clone of the Newsweek homepage ".
* The goal of the project was to learn to develop responsive layouts using bootstrap, flex, grid or float 
* The layout was achieved using the bootstrap grid system 
* The project is responsive and can accommodate small, medium and large screens 
* dummy images and content have been used repeatedly because the focus on this project is responsive design 
* I used semantic HTML to structure the markup

**Comment by trekab** [comment]
# Changes Required!
Great job @chirchir12  :clap:, you have worked on most of the required features. There are a few changes to look at though.

## Required changes:
- [ ] On small screens, your latest news section is very different from the original. ([Your clone](https://github.com/trekab/images/blob/master/clone.png?raw=true), [Original](https://github.com/trekab/images/blob/master/original_site.png?raw=true))

## Optional changes:
- [ ] Make the footer logo the same image you used as the header logo. This will make your site look more like the original.

__Kindly work on the required changes and submit another code review request__
_Feel free to slack me for any inquiries_ :wink: 
__Treasure Kabareebe__

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+  }
+
+  .top-footer-container,
+  .left-content {
+    flex-direction: column;
+  }
+
+  .left-content.align-items-end {
+    margin-bottom: 0.6rem;
+    align-items: flex-start !important;
```
Comment (rammazzoti2000) [inline css/style.css:524]: important property should be avoided at any cost, please remove it and find another way for that property

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hi @chirchir12 , you have a great project :+1:. did great work :tada: :rocket: 
Although there are some other issues, and some of them are crucial, that are preventing me to approve this project.

## Required Changes :construction: :raised_hands: 

- [x]  You should use Bootstrap Css classes everywhere for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png) from [Pathwrigth](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540308/))
 The goal of this project is to use them. You have made heavy use of css classes that could've been easily replaced using bootstrap classes, starting from the basic classes for styling like margins( m-0), paddings(p-0) to more important classes like display (d-block) or position (position:absolute), and so other many classes that can replace css styling
I tried to use inline comments in your css stylesheet to indicate what you need to replace, is just that they are too many, and for this I am asking you please take the time to revisit your stylesheet and determine what can be used with bootstrap and what cannot.
 I know it may be tedious, but that is the essence of this particular project.
#
- [x] The 'Main Story' section should be renamed to 'Top story' [see original website](https://www.newsweek.com/)
#
- [x] The 'Opinion' section on medium size should be properly aligned 
image
#
- [x] Starting with medium size and bellow, your website is having horizontal overflow issues, also should have some margin space on both sides
#
- [x] Latest news on medium size should be properly aligned
image
#
- [x] The Editor's Picks section, between 768px and 1024px, should be aligned on two column
#### Original:
image
#### Clone: 
image
#
- [x] On medium size in the U.S., World, Business, Tech & Business, Culture, Sports , the images on the right are quite tiny
image
#
- [x] The logo in the footer on medium size is quite smaller than the original one, also shoul be the same as the original one
#### Original: 
image
#### Clone:
image
#
- [ ] In css  the use of `!important` property should be avoid at any cost, please remove it and use another way for that property, see the comment bellow
#
- [x] Remove the borders from the articles in `In The Magazine` section
image


### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>

### Status: Changes required.

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Status: Changes Required

**Comment by chirchir12** [comment]
## changes updated accordingly
# note
1. I used most of the css classes that i used could not be replaced by boostrap because they don't give me the exact styling i wanted. 
2. I used bootstrap grid for my layout and there is no place i used css for layout
3. the horizontal scroll bar appears when you have a # in front of the link like this http://127.0.0.1:5500/index.html/# can you help me figure this out. the horizontal scroll is not there when i remove the #
## THANK YOU FOR THE REVIEWS. have a look at it once again

**Comment by sumancrest0001** [comment]
**Status: Changes required** 
Awesome work @chirchir12  :clap:  :clap: :+1: 
Your project looks similar to the original one but there are minor issues that need to be addressed before moving to the next project. Please make the following changes.
**[NOTE] All the changes are mandatory.**

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+
+.bold-font {
+  font-weight: 700;
+  font-size: 1.1rem;
+}
+
+.bold-font:hover {
+  text-decoration: none;
+  color: #fff;
+}
```
Comment (sumancrest0001) [inline css/style.css]: Let's avoid repetition of code and follow the DRY principle. [PathRight requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png). Have a look at this [article](https://speckyboy.com/good-bad-css-practices/) for reference.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+
+.debate-discussions::after {
+  content: '';
+  display: block;
+  background-color: #d9d9d9;
+  height: 1px;
+  position: absolute;
+  top: 15px;
+  left: 0;
+  right: 0;
```
Comment (sumancrest0001) [inline css/style.css]: Let's remove all the repeated codes.

**Comment by sumancrest0001** [comment]
We have to fix the alignment of the newsletter and subscriber sections on large and small screens. On the large screen height of both sections should be equal. Currently, the newsletter has more.
In your project :point_down: 
Screenshot from 2020-05-09 11-57-57

On the original project :point_down: 
Screenshot from 2020-05-09 11-57-41

**Comment by sumancrest0001** [comment]
On the small screen, The newsletter section should be a single column but currently, it is divided into two. 
Have a look :point_down: 
Screenshot from 2020-05-09 12-01-58

It should be like this :point_down: 
Screenshot from 2020-05-09 12-02-16

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Kindly make the above-requested changes and submit the project for another review.

Need help or have questions, feel free to message me on Slack (Suman Shrestha)

Happy coding !!!**

File: css/style.css (hunk 2 above)
Comment (chirchir12) [reply css/style.css]: Hello I managed to figure this out. thanks. however how can i apply same style for different state like :hover ? kindly help

**Comment by chirchir12** [comment]
# I updated all the required changes. kindly review it again

**Review by lucasmazo32 — APPROVED** [review]
# Project approved :rocket:

#### Great job on implementing the required changes for the project.

#### Your project looks great!

### You can go ahead and merge this pull request!

### Happy coding! :computer:

## Lucas Mazo

**Pull request merged**